keras/backend/mxnet_backend.py

```diff
@@ -263,6 +263,14 @@ def variable(value, dtype=None, name=None, constraint=None):
         ret._uses_learning_phase = False
         ret.bind(v)
         return ret
+    elif sparse_weight:
+        v = mx.ndarray.sparse.cast_storage(mx.nd.array(value), 'row_sparse')
```

Comment:
Try creating row_sparse arrays using the row_sparse_array() API or mx.nd.array(value).tostype('row_sparse')
according to the doc:https://mxnet.incubator.apache.org/api/python/ndarray/sparse.html

Reply:
added